cfme/tests/infrastructure/test_provisioning_dialog.py

```diff
@@ -293,3 +293,35 @@ def test_provisioning_schedule(provisioner, provider, prov_data, vm_name):
     template_name = provider.data['provisioning']['template']
 
     provisioner(template_name, prov_data, delayed=provision_time)
+
+
+@pytest.mark.rhv2
+@pytest.mark.uncollectif(lambda provider: not provider.one_of(RHEVMProvider),
+                         reason='These profile names are RHV specific.')
+@pytest.mark.parametrize('vnic_profile', ['<No Profile>', '<Use template nics>'])
+def test_provisioning_vnic_profiles(provisioner, provider, prov_data, vm_name, vnic_profile):
+    """ Tests provision VM with other than specific vnic profile selected - <No Profile>
+        and <Use template nics>.
+
+    Prerequisities:
+        * A provider set up, supporting provisioning in CFME
+
+    Steps:
+        * Open the provisioning dialog.
+        * Apart from the usual provisioning settings, set vlan
+          to values <No Profile>/<Use template nics>
+        * Submit the provisioning request, it should provision the vm successfully.
+
+    Metadata:
+        test_flag: provision
+    """
+    prov_data['catalog']['vm_name'] = vm_name
+    prov_data['network'] = {'vlan': vnic_profile}
+    template_name = provider.data['provisioning']['template']
+
+    vm = provisioner(template_name, prov_data)
+
+    wait_for(
+        lambda: vm.exists_on_provider,
+        num_sec=240, delay=5
+    )
```

Comment:
Double check that `provisioner` is doing cleanup of the vm, if not use `request.addfinalizer` here

Reply:
I added the NIC check only for the 'No Profile' parameter, because checking the 'Use template profile' is problematic - result depends on the template, which may change.

Reply:
@rhrazdil Can we also assert that the NIC has no profile?